jiant/models.py

```diff
@@ -1129,7 +1088,6 @@ def _tagger_forward(self, batch: dict, task: TaggingTask, predict: bool) -> dict
         hid2tag = self._get_classifier(task)
         logits = hid2tag(sent[:, 1:-1, :]).view(b_size * seq_len, -1)
         out["logits"] = logits
-        targs = batch["targs"]["words"][:, :seq_len].contiguous().view(-1)
```

Comment:
Looks like there's a code path below that can throw an exception if this is deleted.